packages/wallets/src/features/cashier/modules/FiatOnRamp/FiatOnRampProvider/FiatOnRampProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import IcCashierApplePay from '../../../../../public/images/ic-cashier-apple-pay.svg';
+import IcCashierBanxa from '../../../../../public/images/ic-cashier-banxa.svg';
+import IcCashierMastercard from '../../../../../public/images/ic-cashier-mastercard.svg';
+import IcCashierVisa from '../../../../../public/images/ic-cashier-visa.svg';
+
+const FiatOnRampProvider = () => ({
```

Comment:
this is not a component, make it as a constants file or move it above the component that use it maybe

Reply:
Changed format to constant file `FiatOnRamp.constants.tsx`